src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -98,6 +98,7 @@ const DesktopArticleVideoPlayer: React.FC<DesktopArticleVideoPlayerProps> = ({ v
 						<JwPlayerWrapper
 							config={getArticleVideoConfig(videoDetails)}
 							onReady={(playerInstance) => articlePlayerOnReady(videoDetails, playerInstance)}
+							pauseOnExitViewport={true}
```

Comment:
This variable makes it sound like it will always pause when it exits the viewport. Which is also a reasonable feature we might do in the future (or at least a/b test). Can we rename it to something more precise?

Reply:
stopAutoAdvanceOnExitViewport might work here.

Reply:
Actually, looking at the behavior the puaseOnExitViewport makes sense. It looks like the player advances to the next video, and then stops/pauses before the video plays. If the user is on another tab while the next video loads, it just pauses.

I think @AlburIvan mentioned that the JW Player behavior for stopping auto advancing only worked on the initial video. This is probably the best compromise we have at the moment to prevent the video from actually playing.

I am curious how this works with ads though. I haven't gotten one on sandbox. My assumption is that the PLAYLIST_ITEM callback gets called when the actual video is about to play, not the ad.